client/src/pages/TurnsHistory/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react'
+
+import { useHouses } from '../../providers/Houses'
+import { useTurns } from '../../hooks/useTurns'
+import { MovieListBackground } from '../../styles/Background'
+import Marquee from '../../components/Marquee'
+import TurnTable from './TurnTable'
+import { ReelImage } from '../../styles/LoadingReel'
+
+const TurnsTable = () => {
```

Comment:
This file is called `TurnsHistory` so the component should be called `TurnsHistory` because you also already have a component called `TurnsTable`